CMakeLists.txt

```diff
@@ -6,6 +6,12 @@ OPTION(USE_HDFS "Enable HDFS support (EXPERIMENTAL)" OFF)
 OPTION(USE_TIMETAG "Set to ON to output time costs" OFF)
 OPTION(USE_CUDA "Enable CUDA-accelerated training (EXPERIMENTAL)" OFF)
 OPTION(USE_DEBUG "Set to ON for Debug mode" OFF)
+OPTION(USE_SANITIZER "Use santizer flags" OFF)
+OPTION(SANITIZER_PATH "Path to sanitizer libs")
+SET(ENABLED_SANITIZERS "address" "leak" "undefined" CACHE STRING
```

Comment:
Can an option be a list? Or should we create a separate option for each sanitizer?

Reply:
I think current behaviour (string with `Semicolon separated list of sanitizer names`) will be just fine.

Reply:
Actually, this has to be `set`, since `option` is meant to be Boolean: https://stackoverflow.com/a/36359003

Reply:
OK, I see. Thanks for taking a try!
I didn't realize that `OPTION` is only a "shortcut syntax" for boolean `SET` with `CACHE`:
 > Since bool cached variables are so common for builds, there is a shortcut syntax for making one using `option`:
    https://hsf-training.github.io/hsf-training-cmake-webpage/05-variables/index.html#cached-variables